core/src/avm1/activation.rs

```diff
@@ -1009,9 +1009,9 @@ impl<'a, 'gc> Activation<'a, 'gc> {
     }
 
     fn action_end_drag(&mut self) -> Result<FrameControl<'gc>, Error<'gc>> {
-        // we might not have had an opportunity to call `update_drag`
-        // if AS did `startDrag(mc);stopDrag();` in one go
-        // so let's do it here
+        // We might not have had an opportunity to call `update_drag`
```

Comment:
nit: This comment appears in two more places - `core/src/avm1/globals/movie_clip.rs` and `core/src/avm2/globals/flash/display/sprite.rs`. Might be worth to update them as well.